indico/modules/events/tracks/models/tracks.py

```diff
@@ -92,6 +97,10 @@ class Track(DescriptionMixin, db.Model):
             lazy=True
         )
     )
+    default_session = db.relationship(
+        'Session',
+        lazy=True
```

Comment:
missing backref

Reply:
@pferreir mentioned it might not be needed, what do you think?

Reply:
omitting the backref is usually not a good idea - AFAIK some cascades only work when there's a backref

Reply:
```python
default_session = db.relationship(
    'Session',
    lazy=True,
    backref=db.backref(
        'assigned_tracks',
        lazy=True
    )
)
```
Something like this?

Reply:
Yes, but I'd call it `default_for_tracks` and you could just use `backref='default_for_tracks'` when using all the defaults anyway (which is the case here)